install/helm/open-match/templates/service-account.yaml

```diff
@@ -13,6 +13,16 @@
 # limitations under the License.
 
 {{- if index .Values "open-match-core" "enabled" }}
+{{- if index .Values "template" }}
```

Comment:
How does the name "template" relate to installing a Namespace?!?

Reply:
We only want to include this namespace when we do `helm template`. Helm2 use namespace to manage its release so `helm install` will be broken if we include this namespace :)

Reply:
Ok, that makes sense, but the name "template" doesn't tell me that.
Maybe name is "usingHelmTemplate"?
Then put a comment explaining this?

Reply:
Done.